README.md

```
@@ -2,18 +2,144 @@
+
+1. Clone the repository
+   ```bash
+   git clone https://github.com/bosankus/Compose-Weatherify.git
+   ```
+
+2. Open the project in Android Studio
+
+3. Get an API key from [OpenWeatherMap](https://openweathermap.org/api)
+
+4. Add your API key to `local.properties`:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)